statsdaemon.go

```diff
@@ -196,7 +210,7 @@ func submit(deadline time.Time) error {
 
 	err = client.SetDeadline(deadline)
 	if err != nil {
-		errmsg := fmt.Sprintf("could not set deadline:", err)
+		errmsg := fmt.Sprint("could not set deadline:", err)
```

Comment:
I know this is a govet cleanup and tangential to this PR but can you just `return err` here?